src/boards/include/boards/wiznet_w5100s_evb_pico.h

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+// -----------------------------------------------------
+// NOTE: THIS HEADER IS ALSO INCLUDED BY ASSEMBLER SO
+//       SHOULD ONLY CONSIST OF PREPROCESSOR DIRECTIVES
+// -----------------------------------------------------
+
+
+
+#ifndef _BOARDS_WIZNET_W5100S_EVB_PICO_H
+#define _BOARDS_WIZNET_W5100S_EVB_PICO_H
+
+// For board detection
+#define WIZNET_W5100S_EVB_PICO
+// --- BOARD SPECIFIC ---
+#ifndef W5100S_EVB_PICO_INTN_PIN
+#define W5100S_EVB_PICO_INTN_PIN 20
```

Comment:
The schematics at https://www.mouser.co.uk/new/wiznet/wiznet-w5100sevbpico-eval-board/ say that INTn is on GPIO21 - so are the schematics wrong, or is the define here wrong? :man_shrugging:

Reply:
Sorry, it is the define wrong.  RSTN and INTN pin numbers were defined opposite to each other.